lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -3921,6 +3921,28 @@ def model_find_and_add_construction(model, climate_zone_set, intended_surface_ty
                                 'building_category' => building_category,
                                 'minimum_percent_of_surface' => minimum_percent_of_surface,
                                 'maximum_percent_of_surface' => maximum_percent_of_surface)
+
+      # 12/14/2021 added to test data pull from sub climate zones for transparent surfaces
+      if !props
+        if climate_zone_set == "ClimateZone 3"
+          # for PRM, climate zone 3 should test on 3A, 3B and 3C for transparent surfaces
```

Comment:
This comment should probably be pushed in the first one but without mentioning PRM (it might already be the case for other climate zone sets).